### PR TITLE
feature(triger): Добавит компоненты Checkbox, Actions, починит селект

### PR DESCRIPTION
**Pull request by dontuse**
https://jira.railsc.ru/browse/GOODS-463

### CODE REVIEW

**Comment by dontuse** [comment]
@Kupov

File: src/styles/mixins/icons.scss [hunk 1]
```diff
@@ -80,6 +80,21 @@
...
+  background: url("data:image/svg+xml;base64,#{$svg-string}") 50%/contain no-repeat;
+}
+
+@mixin trash-o-icon($color: #1f2532) {
+  $svg-string: encode-base64('<svg fill="#{$color}" width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M704 736v576q0 14-9 23t-23 9h-64q-14 0-23-9t-9-23v-576q0-14 9-23t23-9h64q14 0 23 9t9 23zm256 0v576q0 14-9 23t-23 9h-64q-14 0-23-9t-9-23v-576q0-14 9-23t23-9h64q14 0 23 9t9 23zm256 0v576q0 14-9 23t-23 9h-64q-14 0-23-9t-9-23v-576q0-14 9-23t23-9h64q14 0 23 9t9 23zm128 724v-948h-896v948q0 22 7 40.5t14.5 27 10.5 8.5h832q3 0 10.5-8.5t14.5-27 7-40.5zm-672-1076h448l-48-117q-7-9-17-11h-317q-10 2-17 11zm928 32v64q0 14-9 23t-23 9h-96v948q0 83-47 143.5t-113 60.5h-832q-66 0-113-58.5t-47-141.5v-952h-96q-14 0-23-9t-9-23v-64q0-14 9-23t23-9h309l70-167q15-37 54-63t79-26h320q40 0 79 26t54 63l70 167h309q14 0 23 9t9 23z"/></svg>');
+  background: url("data:image/svg+xml;base64,#{$svg-string}") 50%/contain no-repeat;
```
Comment (apress-hound) [inline src/styles/mixins/icons.scss]: `50%/contain` should be written with a single space on each side of the operator: `50% / contain`

File: src/styles/mixins/icons.scss [hunk 2]
```diff
@@ -80,6 +80,21 @@
...
+}
+
+@mixin copy-icon($color: #1f2532) {
+  $svg-string: encode-base64('<svg fill="#{$color}" width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1696 384q40 0 68 28t28 68v1216q0 40-28 68t-68 28h-960q-40 0-68-28t-28-68v-288h-544q-40 0-68-28t-28-68v-672q0-40 20-88t48-76l408-408q28-28 76-48t88-20h416q40 0 68 28t28 68v328q68-40 128-40h416zm-544 213l-299 299h299v-299zm-640-384l-299 299h299v-299zm196 647l316-316v-416h-384v416q0 40-28 68t-68 28h-416v640h512v-256q0-40 20-88t48-76zm956 804v-1152h-384v416q0 40-28 68t-68 28h-416v640h896z"/></svg>');
+  background: url("data:image/svg+xml;base64,#{$svg-string}") 50%/contain no-repeat;
```
Comment (apress-hound) [inline src/styles/mixins/icons.scss]: `50%/contain` should be written with a single space on each side of the operator: `50% / contain`

File: src/styles/mixins/icons.scss [hunk 3]
```diff
@@ -80,6 +80,21 @@
   background: url("data:image/svg+xml;base64,#{$svg-string}") 0%/contain no-repeat;
 }
 
+@mixin plus-square-icon($color: #1f2532) {
+  $svg-string: encode-base64('<svg fill="#{$color}" width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1408 960v-128q0-26-19-45t-45-19h-320v-320q0-26-19-45t-45-19h-128q-26 0-45 19t-19 45v320h-320q-26 0-45 19t-19 45v128q0 26 19 45t45 19h320v320q0 26 19 45t45 19h128q26 0 45-19t19-45v-320h320q26 0 45-19t19-45zm256-544v960q0 119-84.5 203.5t-203.5 84.5h-960q-119 0-203.5-84.5t-84.5-203.5v-960q0-119 84.5-203.5t203.5-84.5h960q119 0 203.5 84.5t84.5 203.5z"/></svg>');
+  background: url("data:image/svg+xml;base64,#{$svg-string}") 50%/contain no-repeat;
```
Comment (apress-hound) [inline src/styles/mixins/icons.scss]: `50%/contain` should be written with a single space on each side of the operator: `50% / contain`

File: src/DropDownMenu/e-dropdown-menu.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  &-title { font-size: 12px; color: #c1c1c1; text-transform: uppercase; margin-bottom: 20px; padding: 0 20px; line-height: 1; }
+  &-menu-item {
+    font-size: 12px; height: 30px; line-height: 30px; padding: 0 20px; cursor: pointer;
+    &:hover { background: #f2f2f2; }
+  }
+  &:after { content: ''; position: absolute; top: -6px; left: 40px; width: 0; height: 0; border-style: solid; border-width: 0 6px 6px; border-color: transparent transparent #fff; }
+  &:before { content: ''; position: absolute; top: -9px; left: 38px; width: 0; height: 0; border-style: solid; border-width: 0 8px 8px; border-color: transparent transparent rgb(221, 221, 224); }
+}
+
+.rc-dropdown-placement-topLeft {
```
Comment (apress-hound) [inline src/DropDownMenu/e-dropdown-menu.scss]: Selector `rc-dropdown-placement-topLeft` should be written in lowercase with hyphens

File: src/ComboSelect/e-select.scss [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
-  * { height: 24px; line-height: 24px; }
-  .rc-select-selection { border-radius: 0; background: #fff; border: solid 1px #e0e0e0!important; font-size: 12px; box-shadow: none !important; }
-  .rc-select-arrow b { @include angle-down-icon(#1f2532); border: none; width: 12px; height: 12px; margin-top: -6px; margin-left: -6px; }
-  .rc-select-enabled .rc-select-selection:hover { border-color: #00a6ff; box-shadow: none; }
+  .rc-select {
+    &-selection,
+    &-selection:hover { border-radius: 0; background: #fff; border: solid 1px #e0e0e0; font-size: 12px; box-shadow: none; }
+    &-arrow b { @include angle-down-icon(#1f2532); border: none; width: 12px; height: 12px; margin-top: -8px; margin-left: -6px; }
+    &-enabled &-selection:hover { border-color: #00a6ff; box-shadow: none; }
+    &-selection--single,
```
Comment (apress-hound) [inline src/ComboSelect/e-select.scss]: Selector `rc-select-selection__rendered` should be written in lowercase with hyphens

**Pull request merged**